src/scheduler/workers/unpublish/weekly/types.ts

```diff
@@ -0,0 +1,20 @@
+import { Address, List, ListItem } from "@prisma/client";
```

Comment:
dejavu 🤔 

Looks the same as `src/scheduler/worker/unpublish/types.ts`. Is there a reason this interface is created in two different places? Well, not exactly the same one is missing `daysUntilUnpublish ?` but  they're similar enough.